src/assets/md/FAQ.md

```diff
@@ -0,0 +1,35 @@
+**What is Mesa?**
+
+Mesa is a decentralized application built on the **Gnosis Protocol**, a decentralized trading protocol for ERC-20s. Instead of using one order book per token pair, the Gnosis Protocol relies on batched auctions and a multidimensional orderbook. These mechanisms enable ring trades, in which liquidity amongst all assets tradable on the protocol is shared. Read more about the protocol’s mechanism on the Gnosis [DevPortal](https://dfusion-docs.dev.gnosisdev.com/docs/introduction1/).
```

Comment:
Can we make the link to open in a new window for every link that it's external?

Reply:
Done